clients/privacy-center/app/server-environment.ts

```diff
@@ -369,6 +371,9 @@ export const loadPrivacyCenterEnvironment =
       FIDES_PRIMARY_COLOR: process.env.FIDES_PRIVACY_CENTER__FIDES_PRIMARY_COLOR
         ? process.env.FIDES_PRIVACY_CENTER__FIDES_PRIMARY_COLOR
         : null,
+      FIDES_CLEAR_COOKIE: process.env.FIDES_CLEAR_COOKIE
+        ? process.env.FIDES_PRIVACY_CENTER__FIDES_PRIMARY_COLOR === "true"
+        : false,
```

Comment:
The environment variable name should be `FIDES_PRIVACY_CENTER__FIDES_CLEAR_COOKIE` to match the current convention
```suggestion
      FIDES_CLEAR_COOKIE: process.env.FIDES_PRIVACY_CENTER__FIDES_CLEAR_COOKIE
        ? process.env.FIDES_PRIVACY_CENTER__FIDES_CLEAR_COOKIE === "true"
        : false,
```